samples/LocalizationSample/Startup.cs

```diff
@@ -10,18 +10,38 @@
 using Microsoft.AspNetCore.Localization;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Options;
+using MyLibrary;
+using OtherNamespace;
 
 namespace LocalizationSample
 {
     public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddLocalization(options => options.ResourcesPath = "My/Resources");
```

Comment:
you need not remove this actually(but yeah remove the `options` thingy here)...if you register the `ClassLibraryStringLocalizerFactory` either before or after this call, it would replace the default `ResourceManagerStringLocalizerFactory`.  This is because we always do a `TryAdd` to avoid overriding user provided implementation: https://github.com/aspnet/Localization/blob/dev/src/Microsoft.Extensions.Localization/LocalizationServiceCollectionExtensions.cs#L64